packages/console/src/components/ColorPicker/index.module.scss

```diff
@@ -0,0 +1,40 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  border-radius: 6px;
+  border: 1px solid var(--color-component-border);
+  outline: 2px solid transparent;
+  transition-property: outline, border;
+  transition-timing-function: ease-in-out;
+  transition-duration: 0.2s;
+  padding: _.unit(2) _.unit(3);
+  font: var(--font-body-medium);
+  position: relative;
+
+  &:focus-within {
+    border-color: var(--color-primary);
+    outline-color: var(--color-primary-80);
+  }
+
+  input {
+    position: absolute;
```

Comment:
why `absolute`?

Reply:
To make it covered by a div, by using this way we can change the apperence of the native color picker icon.

Reply:
Actually we can customize the native color input styles. Check this out:
https://stackoverflow.com/a/65021768/3431443

Reply:
Created LOG-2199